index.md

```diff
@@ -86,6 +86,13 @@ Follow these instructions which will guide you through the setup process:
 
 * [Setup your FeatureBase cloud account](/docs/cloud/cloud-home)
 
+## How do I launch FeatureBase with Docker?
+You can quickly start FeatureBase Community using the [Docker Hub](https://hub.docker.com/r/featurebasedb/featurebase) image:
+
+```
+docker run -p 10101:10101 featurebasedb/featurebase
```

Comment:
I'm confused about the instructions as they're talking about installing FeatureBase from source, not setting up FeatureBase to run with Docker?

Or have I missed something?

Installing FeatureBase from source also features on the application README:
https://github.com/FeatureBaseDB/featurebase#getting-started

Additionally, running the Docker command in Windows Terminal I get this error message:

```
error during connect: In the default daemon configuration on Windows, the docker client must be run with elevated privileges to connect.: Post "http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/images/create?fromImage=featurebasedb%2Ffeaturebase&tag=latest": open //./pipe/docker_engine: The system cannot find the file specified.
```

And this was because there's a missing step > to run Docker Desktop **before** you attempt to run the command.

Reply:
I forgot to mention that even when the command executes successfully there's no FeatureBase app at the stated URL...

Reply:
This URL works for me: https://hub.docker.com/r/featurebasedb/featurebase

That is the Docker Hub page for the image, which is pulled correctly with the given docker command. If someone doesn't have Docker running, that's outside the scope of our support.

This does not build FeatureBase, but pulls an image that then starts it locally, where it can be accessed.

Docker instructions need to go into the instructions, but I'm not going to write them all at once...

Reply:
Additionally, the Docker Hub page is showing the README from the repo and there's no way to change it unless we update the README. I'm OK waiting on this PR until such time that the README on the repo is updated, which needs to be done sooner than later, IMHO.

Reply:
I'm going to argue that there's a difference between an externally managed product that the user may want to setup as they wish (which is outside our scope), and something that's required to make our product work.

For instance:

* some people have Docker set to auto-start, while others don't because it's a bit of a memory hog. 
* The `make` command is another -- not all systems have it installed and it's not obvious that it's an app you need to install **unless you know that beforehand**.

Ultimately we have two types of users of the help:
* People who know what they're doing (e.g., everyone in FeatureBase) who won't necessarily use the help, and if they do, they're more likely to skim the Before you begin because they'll already have their environment setup.
* Customers who are new to the product who need all the help they can get. And if we're explicit about prerequisites and requirements it will make things easier for them. If they know what to do, they can skip the list. If they don't then it's necessary.

Additionally, if something goes wrong, the "Before you begin" serves as a first port-of-call for troubleshooting for users and for support.

Reply:
It is outside of the scope of our community support to manage a user's docker installation, IMHO. However, it is necessary to have docker installed in order to use FeatureBase.

We could add explicit instructions for installing and configuring docker to the help pages. Additionally, a troubleshooting page should be added to address common issues that users may encounter.